pkg/autoscaler/autoscaler.go

```diff
@@ -40,26 +41,35 @@ func NewAutoScaler(parentLogger logger.Logger,
 		groupKind:               options.GroupKind,
 		customMetricsClientSet:  customMetricsClientSet,
 		inScaleToZeroProcessMap: make(map[string]bool),
+		ticker:                  time.NewTicker(options.ScaleInterval.Duration),
```

Comment:
Practically it doesn't matter too much cause class initialization and start called together
but I think it makes more sense to start the ticket in the start and not here